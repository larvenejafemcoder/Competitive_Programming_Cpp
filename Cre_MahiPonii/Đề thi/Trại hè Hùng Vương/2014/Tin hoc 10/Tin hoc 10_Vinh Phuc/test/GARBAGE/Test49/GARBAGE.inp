NUDWNEWEUUNNN
WWDDDESNUUUNSSEEWEUWNNSWDWWDSDDDESEDDEEDDENDSUEUESUSEUUNSDNSSSWESDSUNSWSUENWNSWUDNEWNDEEWEN
NDUWDESWNDENWWSSUDE
WDSDEESNWSSUSDDDESEUENSSUUENWNSNNSWDDWUUWSUUNDEDDWWDUWNESSNNSDSWNUSENNUWESNWDNUSDEE
USSNDSUDUSWWUWWEDUWWWNDDUWSSWUUNWNSSUWSWUDNDUSNNSEENNSEDNWNEEEWSWNNNWNNUWNEUWS
DEUEEWWNUNDUUWSEWWUDNEDWUUEENEDESEEESSDSWDSWEDSWUSSDNDUDWDEWWEDUENEUWSWNWDUDEEEDSWNDSNSUWESSDWNW
D 49
