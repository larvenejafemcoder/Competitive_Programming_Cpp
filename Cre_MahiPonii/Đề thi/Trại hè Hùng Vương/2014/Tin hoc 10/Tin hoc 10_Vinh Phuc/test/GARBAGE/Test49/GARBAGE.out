405890023541681901949841223591328822007899701652378684153927933258067524181675499579061
